### PR TITLE
Make minor changes for 2-3 compatibility and other bug fixes.

### PR DESCRIPTION
**Pull request by pagreene**
Resolve issues found using Eclipse IDE.

### CODE REVIEW

File: indra/resources/entity_hierarchy.rdf [hunk 1]
```diff
@@ -120,6 +120,9 @@
 <http://identifiers.org/hgnc.symbol/APOE> <http://sorger.med.harvard.edu/indra/relations/partof> <http://sorger.med.harvard.edu/indra/entities/HDL> .
 <http://identifiers.org/hgnc.symbol/ARAF> <http://sorger.med.harvard.edu/indra/relations/isa> <http://sorger.med.harvard.edu/indra/entities/RAF> .
 <http://identifiers.org/hgnc.symbol/AREG> <http://sorger.med.harvard.edu/indra/relations/isa> <http://sorger.med.harvard.edu/indra/entities/EGFR_ligand> .
+<http://identifiers.org/hgnc.symbol/ARHGDIA> <http://sorger.med.harvard.edu/indra/relations/isa> <http://sorger.med.harvard.edu/indra/entities/RhoGDI> .
+<http://identifiers.org/hgnc.symbol/ARHGDIB> <http://sorger.med.harvard.edu/indra/relations/isa> <http://sorger.med.harvard.edu/indra/entities/RhoGDI> .
+<http://identifiers.org/hgnc.symbol/ARHGDIG> <http://sorger.med.harvard.edu/indra/relations/isa> <http://sorger.med.harvard.edu/indra/entities/RhoGDI> .
```
Comment (johnbachman) [inline indra/resources/entity_hierarchy.rdf]: Wondering--how did these changes show up?

Comment (pagreene) [inline indra/resources/entity_hierarchy.rdf]: Oh yeah, that has been puzzling me. I thought I excluded them from this transition, but I have no idea how they were created.

Comment (johnbachman) [inline indra/resources/entity_hierarchy.rdf]: Could be that you ran resources/update_resources.py at some point during setup. Will re-run this and check.

File: pysb_model.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# exported from PySB model 'None'
```
Comment (johnbachman) [inline pysb_model.py]: Not sure how this file showed up here (got generated by the PysbAssembler at some point) but we don't need it in version control.

Comment (pagreene) [inline pysb_model.py]: I wasn't sure. It did seem a bit odd to me. Good to know where it came from.

File: indra/resources/entity_hierarchy.rdf (hunk 1 above)
Comment (johnbachman) [inline indra/resources/entity_hierarchy.rdf]: I'm messing with update_resources and running into some issues. Will skip over this one until it's figured out. Probably also has to do with the version of bioentities.

**Comment by johnbachman** [comment]
OK, great! Good finds here. In some cases the errors were picked up in old/out-of-date code, in which case I just got rid of it rather than fix.

**Pull request merged**